README.md

```diff
@@ -173,11 +173,13 @@ In order to speed up testing, Barista keeps in mind some considerations.
 # Download
 
 ```gradle
-androidTestCompile('com.schibsted.spain:barista:1.1.0') {
+androidTestCompile('com.schibsted.spain:barista:x.y.z') {
```

Comment:
I personally hate with all my heart people that don't put the version in the copy&pastable dependency ^^